cabal.project.freeze

```diff
@@ -209,7 +209,7 @@ constraints: any.Boolean ==0.2.4,
              any.old-time ==1.1.0.3,
              any.optparse-applicative ==0.16.1.0,
              optparse-applicative +process,
-             any.pact ==4.2.0,
+             any.pact ==4.2.0.1,

```

Comment:
Not sure if this version needs to go

Reply:
No, it's basically a cw-only release which is fine